string name
geometry_msgs/PoseStamped pose

string TABLE=table
string IMPACT_BATTERY=impact_battery
string MUG_BOX=mugbox
